E3SM-Project/E3SM
# Update CIME to ESMCI cime5.8.7-2

Update CIME to ESMCI cime5.8.7-2

Squash merge of jgfouca/branch-for-to-acme-2019-08-14

Features:
* Update to latest NCAR PIO2 version
* improve the config_compilers.xml schema to catch case errors

Bug fixes:
* make sure output_root path is abspath
* make sure  ~/.cime/config_compilers.xml file is also checked against schema
* fix indentation bug in get_timing

[BFB]
